src/main/java/tools/jackson/databind/cfg/BaseSettings.java

```diff
@@ -136,16 +141,17 @@ public final class BaseSettings
     /**********************************************************************
      */
 
-    public BaseSettings(AnnotationIntrospector ai,
-            PropertyNamingStrategy pns, AccessorNamingStrategy.Provider accNaming,
-            TypeResolverBuilder<?> defaultTyper, PolymorphicTypeValidator ptv,
-            DateFormat dateFormat, HandlerInstantiator hi,
-            Locale locale, TimeZone tz, Base64Variant defaultBase64,
-            CacheProvider cacheProvider, JsonNodeFactory nodeFactory,
-            ConstructorDetector ctorDetector)
+    public BaseSettings(AnnotationIntrospector ai, PropertyNamingStrategy pns,
+                        EnumNamingStrategy ens, AccessorNamingStrategy.Provider accNaming,
```

Comment:
Please don't make source incompatible changes. Add a new constructor and deprecate the old one.